rspotify-model/src/playlist.rs

```diff
@@ -43,7 +43,7 @@ pub struct FullPlaylist {
     pub external_urls: HashMap<String, String>,
     pub followers: Followers,
     pub href: String,
-    pub id: PlaylistId,
+    pub id: PlaylistId<'static>,
```

Comment:
Why do we need to mark the lifetime of `id` as `static`?

Reply:
That's the same as having an owned version. It's just like a `Cow`.